### PR TITLE
Ui Fixes, Nav Restructure

### PR DESCRIPTION
**Pull request by Musimaniac**
This PR includes a number of small fixes to UI and general "shape" of  the website, to prep for first release. Notable changes include moving the Statue tab to the Character Editing Tab (for clarity and because it's functionally tied to the selected char), and grouping calculators togheter with a sub tabbed group

- [x] removed jumbotron for a cleaner look, fixed padding
- [x] statues moved to character tab
- [x] remade build select using built-in input groups
- [x] group Calculators
- [x] write flavour text and user guide on every page

Feedback highly encouraged

### CODE REVIEW

File: src/pages/Characters.vue [hunk 1]
```diff
@@ -163,9 +173,9 @@
             </div>
           </div>
         </div>
+        <statues-section />
```
Comment (adapap) [inline src/pages/Characters.vue:176]: +1 for statues here, fits well.

File: src/pages/PicnicQuest.vue [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <template>
```
Comment (adapap) [inline src/pages/PicnicQuest.vue:1]: Compact picnic quest is on the backburner I suppose? It's NBD for the release

**Review by adapap — APPROVED** [review]
Nice work on the simple tips for each page! I like most of the changes that you made to improve the site, and thanks for adding in the alchemy data. I suggested a few rewording suggestions, up to you if you agree on them.

One smaller thing - I think the character card should show some placeholder for the unnamed characters instead of being blank. It looks a bit confusing. We can make it just "No Name" or similar.

File: src/pages/Calculators.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<template>
+<div class="tabbable">
```
[the remark this answers is not in the document]
Comment (Musimaniac) [reply src/pages/Calculators.vue:2]: yes, I had the same idea, but I liked the option to shamelessly steal... reuse you nav code

File: src/pages/PicnicQuest.vue (hunk 2 above)
Comment (Musimaniac) [reply src/pages/PicnicQuest.vue:1]: yes absolutely, it's a secondary concern

**Pull request merged**